string fileBaseName
string PointCloudFileType
---
bool ok
